Ginger/Ginger/BusinessFlowPages_New/AddActionMenu/ActionsFactory.cs

```diff
@@ -108,7 +108,17 @@ static Act GenerateSelectedAction(Act selectedAction, Context mContext)
             }
             else
             {
-                Act instance = (Act)selectedAction.CreateCopy();
+                Act instance = null;
```

Comment:
Still notice that add Action from Shared Repo action page using Arrow and not D&D is not using ActionsFactory and it should- please modify it on next PR